loader/common/tree_loader.py

```diff
@@ -249,6 +245,14 @@ def _merge_if_child_is_single_internal_node(self, curr_node, ordering):
 
 
 
+
+    def _get_number_of_leaf_descendants(self, curr_node, tree):
+        subtree = dfs_tree(tree, curr_node)
```

Comment:
wonder if you can subvert this by using `sub_nodes = list(nx.descendants(tree, curr_node))`

Reply:
Sure, I'll check this out.